New items, tweaks

zeskorion opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

-several interesting items can now appear in surplus crates
-viro can now buy several different disease cultures. Some, like Spanish Flu and Pierrot's throat, are available to their respective roles.
-Clown can now get clown spawnergrenades, and clownbomb is cheap enough to be worth it
-an insane clown or chaplain can get a superior honkrender for all 20 TC. a clown can get a honkrender for the same price as a clown bomb. Spawns less clowns but is far easier to deploy
-cook can get gluttony's blessing and a veal render for... whatever reason they need them

## Why It's Good For The Game

Adds some fun new items to the game. Viro can buy some fun cultures instead of being incompetent or just flat out killing everyone, clown can get really clowny now, and more!
## Changelog
:cl:
add: Added several new role specific items
tweak: Several new items can appear in surplus crates, albiet rarely
balance: Lowered cost on clown bomb
/:cl:

zeskorion committed:
New items, tweaks

-several interesting items can now appear in surplus crates
-viro can now buy several different disease cultures. Some, like Spanish Flu and Pierrot's throat, are available to their respective roles.
-Clown can now get clown spawnergrenades, and clownbomb is cheap enough to be worth it
-an insane clown or chaplain can get a superior honkrender for all 20 TC. a clown can get a honkrender for the same price as a clown bomb. Spawns less clowns but is far easier to deploy
-cook can get gluttony's blessing and a veal render for... whatever reason they need them

zeskorion committed:
Update uplink_items.dm

ghost commented:
when you say surplus do you mean nullcrates

qwertyquerty commented on code/modules/uplink/uplink_items.dm:
```diff
@@ -1649,6 +1683,62 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
+	name = "Experimental Disease"
+	desc = "A random disease. Maybe you'll get lucky."
+	item = /obj/item/reagent_containers/glass/bottle/random_virus
+	cost = 1
```
Couldn't they just buy a bunch of these and likely get a really good one

zeskorion replied:
No. Higher symptoms are rare to spawn and optimal diseases are rarer

qwertyquerty commented:
Review your prices. I think some of these are too low price or to high of chance for surplus

zeskorion committed:
Update uplink_items.dm

zeskorion commented:
> Review your prices. I think some of these are too low price or to high of chance for surplus

Alright, changed up a few prices. Surplus chances for the big ones, however, are only twice as high as his grace, which is quite low.

> you also shouldnt make miner items available for buying with TC like veal render and glutons blessing, it defeats the point of their lavaland ruins existing

yeah, removin those two

zeskorion committed:
Small tweaks

changed crate percentages. new additions cant appear in nullcrates. romerol is now very rare in surplus

zeskorion commented:
I also made it so the new surplus additions will NOT appear in surplus. Additionally, romerol is now far rarer in surplus

ghost commented:
Surplus isn't a variable Zesk.

zeskorion committed:
Update uplink_items.dm

zeskorion committed:
Update uplink_items.dm

qwertyquerty commented:
What are these ported from

Toxici11i commented:
qwerty all of these are already in code, maybe except the honkrender one but thats just modified

Toxici11i commented:
they were never added to the uplink though lmao

zeskorion commented:
> What are these ported from

what toxic said. those like the CLUWNE grenades look like they may've been intended for the uplink but just never added.

qwertyquerty commented on code/modules/uplink/uplink_items.dm:
```diff
@@ -1606,9 +1608,27 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			be defused, and some crew may attempt to do so."
 	item = /obj/item/sbeacondrop/clownbomb
-	cost = 15
+	cost = 8
```
This seems super cheap

qwertyquerty commented on code/modules/uplink/uplink_items.dm:
```diff
@@ -576,6 +578,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			along with slurred speech, aggression, and the ability to infect others with this agent."
 	item = /obj/item/storage/box/syndie_kit/romerol
 	cost = 28
+	surplus = 10
```
This seems too high considering the price

zeskorion replied:
Note that these are not percentage chances of appearing. I think they're the percentage chance it is included in the pool at all. It used to be ALWAYS included

qwertyquerty commented on code/modules/uplink/uplink_items.dm:
```diff
@@ -537,7 +538,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/book/granter/martial/carp
 	cost = 16
-	surplus = 0
+	surplus = 10
```
Too high in my opinion

zeskorion replied:
seems high in paper but testing shows it's extremely rare. Especially considering how much it costs for a surplus crate and how infrequently they are picked

qwertyquerty replied:
We don't want surplus crates to become the new meta either.

qwertyquerty commented on code/modules/uplink/uplink_items.dm:
```diff
@@ -401,7 +401,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/box/syndie_kit/guardian
 	cost = 18
-	surplus = 0
+	surplus = 10
```
Way too high in my opinion

zeskorion replied:
ditto

qwertyquerty committed:
Merge branch 'master' into uplink

qwertyquerty committed:
fix price
